addons/survey/report/survey_templates.xml

```diff
@@ -56,7 +56,7 @@
                             <span>Date</span>
                         </div>
                         <div class="certification-company">
-                            <span class="certification-company-logo" t-field="user_input.create_uid.company_id.logo" t-options="{'widget': 'image'}" role="img"/>
+                            <span class="certification-company-logo" t-field="user_input.create_uid.company_id.sudo().logo" t-options="{'widget': 'image'}" role="img"/>
```

Comment:
Here, the same thing as above but also, in my tests I couldn't get the image through `.logo`, correct me if I'm wrong and it works for you, alternatively we could use `partner_id.image_1920` but could be interesting to investigate why the image widget does not process the logo with sudo.

```suggestion
                            <span class="certification-company-logo" t-field="user_input.create_uid.sudo().company_id.logo" t-options="{'widget': 'image'}" role="img"/>
```